src/frontend/components/AppRoutes/index.tsx

```diff
@@ -26,7 +25,7 @@ export const AppRoutes = () => (
       <Switch>
         <Route
```

Comment:
You can go one step further and only specify one route here that renders differently based on `match.params.objectType`

Reply:
:ok_hand: